docs/pages/guides/using-bun.mdx

```diff
@@ -31,7 +33,9 @@ EAS decides which package manager to use based on the lockfile in your codebase.
 
 ### Customize Bun version on EAS
 
-EAS uses `bun@1.0.14` by default. If you need to use a particular version of Bun, you can configure the exact version in each build in your **eas.json**:
+Bun is installed by default when using EAS. To check the default version of Bun when creating an EAS Build, see [Android latest server image](/build-reference/infrastructure/#details) and [iOS latest server image](/build-reference/infrastructure/#details-7).
+
+If you need to use a particular version of Bun, you can configure the exact version in each build in your **eas.json**:
```

Comment:
I noticed that we don't have docs for `bun` option in [eas.json reference](https://docs.expo.dev/eas/json). Can we add it and link to `/eas/json/#bun` from here?

Reply:
Good point. Yeah, I'll update the **eas.json** refernece and link it here.

Reply:
@szdziedzic, I've add `bun` in eas.json schema, added a link of it here too.